### PR TITLE
JI-4380 add missing focus effects and update existing one. refactor b…

### PR DESCRIPTION
**Pull request by Gwenillia**
…uttons

### CODE REVIEW

File: src/components/generic/keywords/Keywords.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ const Keywords = ({ chips, setKeywords, setTempKeywords, id }) => {
         />
         <button
           disabled={inputValue.length < 1}
-          className={`h5p-hub-add-button ${inputValue.length < 1 ? 'disabled' : ''}`}
+          className={`btn btn--add`}
           onClick={handleButtonClick}
           dangerouslySetInnerHTML={{ __html: `+ ${l10n.add}` }}
           onFocus={() => setButtonFocus(true)}
```
Comment (Gwenillia) [inline src/components/generic/keywords/Keywords.js:141]: Should be a to do a <Button />

File: src/components/generic/selector/disciplineElement/DisciplineElement.js [hunk 2]
```diff
@@ -57,6 +57,7 @@ const DisciplineElement = React.forwardRef(({
       onKeyDown={handleKeyDown}>
       <div className='h5p-hub-content' key={'label' + id}>
         <button
+          className='btn btn--add'
           onClick={(e) =>
             onCheckedClick(filter, id, !checked, e)
           }
```
Comment (Gwenillia) [inline src/components/generic/selector/disciplineElement/DisciplineElement.js:63]: should be a <Button />

File: src/components/generic/button/Button.js [hunk 3]
```diff
@@ -3,22 +3,30 @@ import PropTypes from 'prop-types';
 
 import './Button.scss';
 
-const Button = ({children, name, onClick, variant, color, enabled, id}) => {
+const Button = ({children, name, onClick, variant, color, enabled, id, ref}) => {
```
Comment (Gwenillia) [inline src/components/generic/button/Button.js:6]: We also could render a variant in this file like <LinkedButton /> which would render an <a> element with some property (would rename Button.js into Buttons.Js(x)

both could be exported and then imported like so : {LinkedButton (or Button)} from 'Buttons.js'

may be overkill for now as there's only one linked button 😄 so I didn't do it yet

File: src/components/generic/form/ImageUpload.scss [hunk 4]
```diff
@@ -74,9 +73,13 @@
...
       }
 
+      &:focus {
+        color: #186df7;
+      }
+
       &:before {
         content: "+";
-        font-size: 2em;
+        font-size: 2.5rem;
```
Comment (Gwenillia) [inline src/components/generic/form/ImageUpload.scss:82]: 2.5 rem will look buggy on this repo. But it's meant to fit the h5pcom repo units (could enforce px but probably then should enforce px everywhere 🤔)

File: src/components/generic/button/Button.scss [hunk 5]
```diff
@@ -1,54 +1,72 @@
...
-    color: #757575;
-    border-radius: 3em;
-    border: 1px solid #d6d6d6;
-    cursor: pointer;
-    font-weight: bold;
-
-    &:before {
-      font-weight: normal;
-    }
+.btn {
```
Comment (Languafe) [inline src/components/generic/button/Button.scss:1]: Can you please explain why you have changed the selector here? This seems like it is very unspecific and will potentially affect other buttons on the page which we don't intend to change.

The previous selector ensured the rules within would not affect anything outside the scope of ".h5p-hub-publish" or ".h5p-hub-registration".

Comment (Gwenillia) [reply src/components/generic/button/Button.scss:1]: I'm okay with it :)

**Pull request merged**